pkg/auth/rights/auth_info.go

```diff
@@ -47,3 +48,22 @@ func AuthInfo(ctx context.Context) (authInfo *ttnpb.AuthInfoResponse, err error)
 	}
 	return authInfo, nil
 }
+
+var errUnauthenticated = errors.DefineUnauthenticated("unauthenticated", "unauthenticated")
+
+// RequireAuthentication confirms if the authentication information within a context contains any rights, if so,
+// the user is considered to be authenticated.
+func RequireAuthentication(ctx context.Context) error {
+	log.FromContext(ctx).Debug("Validating request authentication")
```

Comment:
Thou shalt use imperative. Also you can simply log `Authenticate request`.

Reply:
I thank thee for noticing this mistake.